var/spack/repos/builtin/packages/libnetworkit/package.py

```diff
@@ -22,6 +22,7 @@ class Libnetworkit(CMakePackage):
 
     maintainers = ['fabratu']
```

Comment:
Want to be a maintainer for `py-networkit` too?

Reply:
Yes, added it.